Functions/Initialization/xASL_init_Iteration.m

```diff
@@ -114,8 +114,11 @@
         dbSettings.diaryFile = fullfile('<ROOT>', 'derivatives', 'ExploreASL' ,'log', [moduleName '_sub-<SUBJECT>.log']);
     elseif ~isempty(regexpi(ModName,'(Structural|LongReg)', 'once'))
         dbSettings.diaryFile = fullfile('<ROOT>', 'log', [moduleName '_<SUBJECT>.log']);
+    elseif ~isempty(regexpi(ModName,'(ASL|func|dwi)', 'once'))        
```

Comment:
We don't want the local log? There's another place where this is done?

Reply:
I don't understand your question. I have only done the following here:

1. Ensure that structural & LongRun don't get <SESSION> in their log filename (because they don't have SESSIONS, and it crashed because SESSIONS weren't defined)
2. Ensure that ASL|func|dwi do get SESSIONS in the log filename.
3. Use `fullfile` instead of `/`

I didn't change any location?

Reply:
Sorry. I don't understand this neither anymore :) This what you mention is all fine. I just don't understand the lockdir change for DARTEL+LongReg - see the other comment.